### DISCUSSION TITLE
Github API Get Organization Creation Date

### DISCUSSION THREAD

**Discussion by benstear** [community/community#24544] [Apps, API and Webhooks]
<p>Hi,</p>
<p>I am trying to find the creation date for all organizations that have a specific word in their name or description, for example, ‘biology’.  So far I have something like this</p>
<pre><code>curl https://api.github.com/search/users?q=biology+type:org
</code></pre>
<p>Which gives me objects like this…</p>
<pre><code>{
      "login": "computational-biology",
      "id": 7949613,
      "node_id": "MDEyOk9yZ2FuaXphdGlvbjc5NDk2MTM=",
      "avatar_url": "https://avatars2.githubusercontent.com/u/7949613?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/computational-biology",
      "html_url": "https://github.com/computational-biology",
      "followers_url": "https://api.github.com/users/computational-biology/followers",
      "following_url": "https://api.github.com/users/computational-biology/following{/other_user}",
      "gists_url": "https://api.github.com/users/computational-biology/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/computational-biology/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/computational-biology/subscriptions",
      "organizations_url": "https://api.github.com/users/computational-biology/orgs",
      "repos_url": "https://api.github.com/users/computational-biology/repos",
      "events_url": "https://api.github.com/users/computational-biology/events{/privacy}",
      "received_events_url": "https://api.github.com/users/computational-biology/received_events",
      "type": "Organization",
      "site_admin": false,
      "score": 1.0
    }
</code></pre>
<p>I can go to the ‘repos_url’ key and get the creation date of the individual repos, but is there any way to simply get the creation date of the organization itself?</p>
<p>Thank you.</p>

**Comment by parithiban** [comment, marked as answer]
<p><a class="mention" href="/benstear">@benstear</a> </p>
<p>You can get the organization created date by calling the organization API.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/2/29b264fb7c0687827b9f90d638c8838872995907.png" data-download-href="/uploads/short-url/5WRMEeKwy68uKhAYvBCPUtoJzv1.png?dl=1" title="Screen Shot 2020-05-16 at 7.07.15 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/7281166/181085234-f51db24e-5032-433a-97c5-083f95786afd.png" alt="Screen Shot 2020-05-16 at 7.07.15 PM.png" data-base62-sha1="5WRMEeKwy68uKhAYvBCPUtoJzv1" width="690" height="407" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/2/29b264fb7c0687827b9f90d638c8838872995907_2_690x407.png, https://aws1.discourse-cdn.com/github/optimized/2X/2/29b264fb7c0687827b9f90d638c8838872995907_2_1035x610.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/2/29b264fb7c0687827b9f90d638c8838872995907_2_1380x814.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/29b264fb7c0687827b9f90d638c8838872995907_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-05-16 at 7.07.15 PM.png</span><span class="informations">2244×1326 411 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by dulcelopez5723-dotcom** [comment]
> Hola
> 
> Estoy tratando de encontrar la fecha de creación de todas las organizaciones que tienen una palabra específica en su nombre o descripción, por ejemplo, 'biología'. Hasta ahora tengo algo como esto
> 
> ```
> curl https://api.github.com/search/users?q=biology+type:org
> ```
> 
> Lo que me da objetos como este...
> 
> ```
> {
>       "login": "computational-biology",
>       "id": 7949613,
>       "node_id": "MDEyOk9yZ2FuaXphdGlvbjc5NDk2MTM=",
>       "avatar_url": "https://avatars2.githubusercontent.com/u/7949613?v=4",
>       "gravatar_id": "",
>       "url": "https://api.github.com/users/computational-biology",
>       "html_url": "https://github.com/computational-biology",
>       "followers_url": "https://api.github.com/users/computational-biology/followers",
>       "following_url": "https://api.github.com/users/computational-biology/following{/other_user}",
>       "gists_url": "https://api.github.com/users/computational-biology/gists{/gist_id}",
>       "starred_url": "https://api.github.com/users/computational-biology/starred{/owner}{/repo}",
>       "subscriptions_url": "https://api.github.com/users/computational-biology/subscriptions",
>       "organizations_url": "https://api.github.com/users/computational-biology/orgs",
>       "repos_url": "https://api.github.com/users/computational-biology/repos",
>       "events_url": "https://api.github.com/users/computational-biology/events{/privacy}",
>       "received_events_url": "https://api.github.com/users/computational-biology/received_events",
>       "type": "Organization",
>       "site_admin": false,
>       "score": 1.0
>     }
> ```
> 
> Puedo ir a la tecla 'repos_url' y obtener la fecha de creación de los repositorios individuales, pero ¿hay alguna forma de simplemente obtener la fecha de creación de la organización en sí?
> 
> Gracias.

Ok
